utils/sane-backends/patches/040-fix-saned-foreground-for-procd.patch

```diff
@@ -0,0 +1,69 @@
+Index: sane-backends-1.0.24/frontend/saned.c
```

Comment:
The package is using sane 1.0.25 while you mention 1.0.24. I worked with upstream project in order to reduce patches to a minimum. The next version will incorporate most of the "fix" patches.

Please, submit this upstream. They are very welcoming. When applied upstream, It would be interesting to refer it on the patch header. This helps when upstream releases a new version and I need to check every patch in order to check if it is needed or not.